### PR TITLE
Add support for running Fathom in client-side applications on older browsers

### PR DESCRIPTION
**Pull request by sleroux**
This patch brings in Babel and Webpack to transpile the existing Fathom code from ES6 -> ES5 for use in older/mobile browsers which lack support for newer JS features. Along with transpiling, dependencies are inlined using webpack to allow including a single JS file using a <script> tag.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,3 +95,12 @@ Finally, "yank" out interesting nodes based on their flavors and scores. For exa
...
 
 Our docs are a little sparse so far, but [our tests](https://github.com/mozilla/fathom/tree/master/test) might help you in the meantime.
+
+## Running Fathom on the Client
+
+Out of the box, Fathom assumes your running the library in a Node.js project. If you're interested in running Fathom on the client side, you can run:
+
+`npm run clientize`
+
+This command will create a file named _fathom.bundle.js_ under the bin/ directory that you can import into your web page using a <script> tag.
```
Comment (pdehaan) [inline README.md]: the `<script>` tag needs to be wrapped in backticks/code format, since it seems to be swallowing the rendering of the rest of the page.

Comment (sleroux) [inline README.md]: Thanks! I've wrapped it in back ticks.

**Comment by erikrose** [comment]
Omg, you figured out webpack. That's better than I've ever managed. ;-) Thanks! I'll have a look at this later.

File: README.md [hunk 2]
```diff
@@ -95,3 +95,12 @@ Finally, "yank" out interesting nodes based on their flavors and scores. For exa
 ## More Examples
 
 Our docs are a little sparse so far, but [our tests](https://github.com/mozilla/fathom/tree/master/test) might help you in the meantime.
+
+## Running Fathom on the Client
+
+Out of the box, Fathom assumes your running the library in a Node.js project. If you're interested in running Fathom on the client side, you can run:
```
Comment (rnewman) [inline README.md]: s/your/you are

File: package.json [hunk 3]
```diff
@@ -10,11 +10,16 @@
     "wu": "^2.1.0"
   },
   "devDependencies": {
+    "babel": "^6.5.2",
+    "babel-cli": "^6.10.1",
+    "babel-loader": "^6.2.4",
+    "babel-preset-es2015": "^6.9.0",
```
Comment (rnewman) [inline package.json]: You probably want to pin these versions, no?

File: package.json [hunk 4]
```diff
@@ -10,11 +10,16 @@
     "wu": "^2.1.0"
   },
   "devDependencies": {
+    "babel": "^6.5.2",
+    "babel-cli": "^6.10.1",
```
Comment (rnewman) [inline package.json]: I don't think you need `babel-cli`. Just `babel-core`, `babel-loader`, `babel-polyfill`, `babel-preset-es2015`.

File: package.json [hunk 5]
```diff
@@ -34,6 +39,7 @@
   "scripts": {
     "lint": "make lint",
     "pretest": "npm run lint",
-    "test": "mocha"
+    "test": "mocha",
+    "clientize": "webpack"
```
Comment (rnewman) [inline package.json]: If you make this `"webpack --optimize-minimize --optimize-dedupe"` you'll save about 300KB in the output.

**Comment by erikrose** [comment]
We've since switched to rollup, and ES5 is just a memory. Sorry for the long neglect on this.

**Pull request closed without merging**